shared/src/packages/chats/libs/validation-schemas/create-chat.validation-schema.ts

```diff
@@ -0,0 +1,13 @@
+import joi from 'joi';
+
+import { ChatValidationMessage } from '../enums/enums.js';
+import { type ChatRequestDto } from '../types/types.js';
+
+const createChat = joi.object<ChatRequestDto, true>({
+  name: joi.string().trim().required().messages({
+    'any.required': ChatValidationMessage.NAME_REQUIRED,
+    'string.empty': ChatValidationMessage.NAME_REQUIRED,
+  }),
+});
```

Comment:
we will generate name on the backend side, so for now just set default name with chat id in backend service like `Chat #12`
```suggestion
```

Reply:
no problem

Reply:
I thought we would generate a name by sending another request and it would depend on the user's message

Reply:
i think we will send first message with the chat creation, because we can't create messages before chat is created and we can't add chat name without message, so update your logic to this please, but don't do something with this message for now

Reply:
how about creating an endpoint like `/first-message` to handle this situation?

Reply:
this doesn't fix the situation, because we can't create a message without chat, so I have two options:

1. send request to generate name without creating a message, then request to create chat, then request to create message - 3 requests needed (will take too much time for the first message)
2. send request to create chat with the first message, generate name on backend, create chat, create message - everything in one request

Reply:
I see. Then we would choose the second option, and in this case, I need to add a message to the create chat request body

Reply:
and creating a message would be on @Xxandder in this issue #190